src/tests/ftest/control/dmg_storage_query.py

```diff
@@ -117,37 +129,28 @@ def test_dmg_storage_query_device_health(self):
         :avocado: tags=control,dmg,storage_query,basic
         :avocado: tags=DmgStorageQuery,test_dmg_storage_query_device_health
         """
-        dmg_info = self.get_device_info(health=True)
-
-        # Cleanup output
-        if dmg_info:
-            for index, info in enumerate(dmg_info):
-                dmg_info[index] = [entry for entry in info if entry]
-        parsed = [dmg_info[index:(index + 19)] for index in range(0, len(dmg_info), 19)]
-        _ = parsed[0].pop(0)
-
-        # Convert from list of lists to list of strings
-        health_info = []
-        for index in parsed:
-            health_info.append([elem[0] for elem in index])
-
-        self.log.info("Found health info: %s", str(health_info))
-
-        # Verify temperature, convert from Kelvins to Celsius
-        temp_err = []
-        for info in health_info:
-            cels_temp = int("".join(re.findall(r"\d+", info[2]))) - 273.15
-            if not 0.00 <= cels_temp <= 71.00:
-                temp_err.append("{}".format(cels_temp))
-        if temp_err:
-            self.fail("Bad temperature on SSDs: {}".format(",".join(temp_err)))
+        errors = []
+        device_info = get_storage_query_device_info(self, self.dmg, health=True)
+        for device in device_info:
+            self.log.info("Health Info for %s:", device['uuid'])
+            for key in sorted(device['health']):
```

Comment:
FYI that `sorted` works on `dict.items()`
```suggestion
            for key, val in sorted(device['health'].items()):
```